src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -173,7 +173,7 @@ def default_api_version(self):
             'role_definitions': '2018-01-01-preview',
             'provider_operations_metadata': '2018-01-01-preview'
         }),
-        ResourceType.MGMT_CONTAINERREGISTRY: SDKProfile('2021-06-01-preview', {
+        ResourceType.MGMT_CONTAINERREGISTRY: SDKProfile('2021-08-01-preview', {
```

Comment:
The failure of CI is caused by the api-version of `ResourceType.MGMT_CONTAINERREGISTRY` upgraded from `2021-06-01-preview` to `2021-08-01-preview` and needed to re record these tests in live mode

>           raise AssertionError(ex)
E           AssertionError: Can't overwrite existing cassette ('/home/vsts/work/1/s/src/azure-cli/azure/cli/command_modules/appservice/tests/latest/recordings/test_acr_integration.yaml') in your current record mode ('once').
E           No match for the request (<Request (PUT) https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.ContainerRegistry/registries/webappacrtest000003?api-version=2021-08-01-preview>) was found.
E           Found 1 similar requests with 1 different matcher(s) :
E           
E           1 - (<Request (PUT) https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.ContainerRegistry/registries/webappacrtest000003?api-version=2021-06-01-preview>).
E           Matchers succeeded : ['method', 'scheme', 'host', 'port', 'path']
E           Matchers failed :
E           _custom_request_query_matcher - assertion failure :
E           None

src/azure-cli-testsdk/azure/cli/testsdk/base.py:299: AssertionError
- generated xml file: /home/vsts/.azdev/env_config/home/vsts/work/1/s/env/test_results.xml -
> =========================== short test summary info ============================
FAILED src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py::TagScenarioTest::test_tag_update_by_patch
FAILED src/azure-cli/azure/cli/command_modules/network/tests/latest/test_private_endpoint_commands.py::NetworkPrivateLinkACRScenarioTest::test_private_endpoint_connection_acr
FAILED src/azure-cli/azure/cli/command_modules/network/tests/latest/test_private_endpoint_commands.py::NetworkPrivateLinkACRScenarioTest::test_private_link_resource_acr
FAILED src/azure-cli/azure/cli/command_modules/acr/tests/latest/test_acr_commands.py::AcrCommandsTests::test_acr_image_import
FAILED src/azure-cli/azure/cli/command_modules/acr/tests/latest/test_acr_connectedregistry_commands.py::AcrConnectedRegistryCommandsTests::test_acr_connectedregistry
FAILED src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_functionapp_commands.py::FunctionappACRScenarioTest::test_acr_integration_function_app
FAILED src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_functionapp_commands.py::FunctionAppCreateUsingACR::test_acr_create_function_app
FAILED src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_functionapp_commands.py::FunctionappACRDeploymentScenarioTest::test_acr_deployment_function_app
FAILED src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py::WebappACRScenarioTest::test_acr_integration
=========== 9 failed, 2693 passed, 314 skipped in 3579.93s (0:59:39) ===========